src/commands/miscellaneous/member.ts

```diff
@@ -0,0 +1,104 @@
+import { Command, container } from '@sapphire/framework';
```

Comment:
Put the details in the `commandDetails` folder and the codey command in the `commands` folder to make it consistent with the rest of the code.